core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
@seanlip . I'm not completely sure about how the VM works, but this is what I think happens. The creator hits submit and the TrainClassiferJobModel is sent to the VM. I think the creator can continue after this is sent. The training happens independent of the creator. @prasanna08 Could you verify if this is how it actually works? This is what I understood from our discussion today morning.